commands/add-key.js

```diff
@@ -0,0 +1,43 @@
+const exitOnError = require('../utils/exit-on-error');
+const connect = require('../utils/connect');
+const inspectResponse = require('../utils/inspect-response');
+const { utils } = require('near-api-js');
+
+module.exports = {
+    command: 'add-key [access-key]',
+    desc: 'Add an access key to given account',
+    builder: (yargs) => yargs
+        .option('access-key', {
+            desc: 'Public key to add (base58 encoded)',
+            type: 'string',
+            required: true,
+        })
+        .option('contract-id', {
+            desc: 'Limit access key to given contract (if not provided - will create full access key)',
+            type: 'string',
+            required: false,
+        })
+        .option('method-names', {
+            desc: 'Method names to limit access key to (command separated)',
+            type: 'string',
+            required: false,
+        })
+        .option('allowance', {
+            desc: 'Allowance in $NEAR for the key (default 0)',
+            type: 'string',
+            required: false,
+        }),
+    handler: exitOnError(addAccessKey)
+};
+
+async function addAccessKey(options) {
+    if (!options.accountId) {
```

Comment:
@ilblackdragon shouldn't this throw error?

Reply:
I'll change to throw error unless there are any objections.

Reply:
or maybe just change to `add-key <account-id> <access-key>` as both seem to be require arguments

Reply:
I just copied this from delete-key -> so prob should change there too.